docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ReplaceMissingValuesMultiColumn.cs

```diff
@@ -0,0 +1,119 @@
+            var data = mlContext.Data.LoadFromEnumerable(samples);
+
+            // ReplaceMissingValues is used to create a column where missing values are replaced according to the ReplacementMode.
+            var meanPipeline = mlContext.Transforms.ReplaceMissingValues(new[] {
+                new InputOutputColumnPair("MissingReplaced1", "Features1"),
+                new InputOutputColumnPair("MissingReplaced2", "Features2")
+            },
+            MissingValueReplacingEstimator.ReplacementMode.Mean);
+
+            // Now we can transform the data and look at the output to confirm the behavior of the estimator.
+            // This operation doesn't actually evaluate data until we read the data below.
+            var meanTransformer = meanPipeline.Fit(data);
+            var meanTransformedData = meanTransformer.Transform(data);
+
+            // We can extract the newly created column as an IEnumerable of SampleDataTransformed, the class we define below.
+            var meanRowEnumerable = mlContext.Data.CreateEnumerable<SampleDataTransformed>(meanTransformedData, reuseRowObject: false);
+
+            // ReplaceMissingValues is used to create a column where missing values are replaced according to the ReplacementMode.
+            var defaultPipeline = mlContext.Transforms.ReplaceMissingValues(new[] {
+                new InputOutputColumnPair("MissingReplaced1", "Features1"),
+                new InputOutputColumnPair("MissingReplaced2", "Features2")
+            },
+            MissingValueReplacingEstimator.ReplacementMode.DefaultValue);
+
+            // Now we can transform the data and look at the output to confirm the behavior of the estimator.
+            // This operation doesn't actually evaluate data until we read the data below.
+            var defaultTransformer = defaultPipeline.Fit(data);
+            var defaultTransformedData = defaultTransformer.Transform(data);
+
+            // We can extract the newly created column as an IEnumerable of SampleDataTransformed, the class we define below.
+            var defaultRowEnumerable = mlContext.Data.CreateEnumerable<SampleDataTransformed>(defaultTransformedData, reuseRowObject: false);
+
+            // a small printing utility
+            Func<object[], string> vectorPrinter = (object[] vector) =>
+            {
+                string preview = "[";
+                foreach (var slot in vector)
+                    preview += $"{slot} ";
+                return preview += "]";
+
+            };
+
+            // And finally, we can write out the rows of the dataset, looking at the columns of interest.
+            foreach (var row in meanRowEnumerable)
+            {
+                Console.WriteLine($"Label: {row.Label} Features1: {vectorPrinter(row.Features1.Cast<object>().ToArray())}" +
+                    $" Features2: {vectorPrinter(row.Features2.Cast<object>().ToArray())} " +
```

Comment:
>Cast<object> [](start = 63, length = 12)

this Cast is only required for printing, but it's making the sample look complicated. please push it to the helper print function. also the print function can take float[], can't it?